resources/crm-local-develop-environment/localstack-init/setup-aws-services.sh

```
@@ -0,0 +1,315 @@
+    --targets "Id"="1","Arn"="$SES_SQS_ARN"
+
+echo "✅ EventBridge targets configured"
+
+# Test EventBridge by sending a sample event
+echo "🧪 Testing EventBridge with sample events..."
+awslocal events put-events \
+    --entries '[
+        {
+            "Source": "local.user",
+            "DetailType": "User Registration", 
+            "Detail": "{\"userId\":\"test-001\",\"email\":\"test@example.com\",\"timestamp\":\"'$(date -u +%Y-%m-%dT%H:%M:%SZ)'\"}"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)